src/plugins/add-cal/reports/a11y-1-fr.hbs

```diff
@@ -0,0 +1,75 @@
+---
+{
+	"title": "Add to Calendar",
+	"language": "fr",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
+	"tag": "add-cal",
+	"parentdir": "add-cal",
+	"altLangPage": "a11y-1-fr.html",
+	"dateModified": "2022-06-22"
+}
+---
+<div id="ds-a11y" data-wb-jsonmanager='{ "url": "a11y-1.json", "name": "a11y" }'></div>
+
+<dl class="dl-horizontal">
+	<dt>Identifiant du rapport d'évaluation</dt>
+	<dd data-json-replace="#[a11y]/id" lang="en"><span lang="fr">Non disponible</span></dd>
+	<dt>Type</dt>
+	<dd>Évaluation de l'accessibilité</dd>
+	<dt>Affirmé par:</dt>
+	<dd data-json-replace="#[a11y]/assertedBy" lang="en"><span lang="fr">Non disponible</span></dd>
+	<dt>Date:</dt>
+	<dd data-json-replace="#[a11y]/date" lang="en"><span lang="fr">Non disponible</span></dd>
+	<dt>Page:</dt>
+	<dd><a data-wb-json='[
+
+			{
+				"url": "#[a11y]/subjectUrl",
+				"type": "attr",
+				"attr": "href"
+			},
+			{
+				"url": "#[a11y]/subjectUrl",
+				"type": "replace"
+			}
+
+		]' href="#" lang="en"><span lang="fr">Non Définie</span></a></dd>
```

Comment:
Keep the `href` attribute empty instead of being an anchor on the current page. If this cause an html lint error, then let's add an exception for our reports.

```suggestion
		]' href lang="en"><span lang="fr">Non Définie</span></a></dd>
```

Reply:
Commenting to resolve conversation, changes were applied in previous commit